### PR TITLE
Subgrounds Quickstart Cookbook added

### PR DESCRIPTION
**Pull request by marcusrein**

### CODE REVIEW

File: website/pages/en/cookbook/subgrounds-quickstart.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+pip install --upgrade subgrounds
+# or
+python -m pip install --upgrade subgrounds
+```
+
+Note that subgrounds requires Python 3.10 or higher and is available on [pypi](https://pypi.org/project/subgrounds/).
+
+## Step 2: Import Subgrounds and Set Your API Key
+
+Subgrounds allows interfacing with all subgraph endpoints, including both subgraphs in [Subgraph Studio](https://thegraph.com/studio/apikeys/) as well as subgraphs published to The Graph Network. To begin using subgrounds with The Graph Network, import the library and make sure you have a valid API key. You can generate an API key in Subgraph Studio to use it inside the subgraph’s query URL as shown [here](https://thegraph.com/docs/en/querying/querying-the-graph/#using-the-graph-explorer), or create a free Playgrounds API key. Currently Playgrounds offers 5,000 free queries per month.
```
Comment (benface) [inline website/pages/en/cookbook/subgrounds-quickstart.mdx:29]:
```suggestion
Subgrounds allows interfacing with all subgraph endpoints, including both subgraphs in [Subgraph Studio](https://thegraph.com/studio/apikeys/) as well as subgraphs published to The Graph Network. To begin using subgrounds with The Graph Network, import the library and make sure you have a valid API key. You can generate an API key in Subgraph Studio to use it inside the subgraph’s query URL as shown [here](/querying/querying-the-graph/#using-the-graph-explorer), or create a free Playgrounds API key. Currently Playgrounds offers 5,000 free queries per month.
```
Relative links ensure that users clicking on them in a translated version of the page don't land on an English page.

We should really have a lint rule for that. 😛 cc @dimaMachina @saihaj

Comment (benface) [reply website/pages/en/cookbook/subgrounds-quickstart.mdx:29]: Though now that I think about it, the `#using-the-graph-explorer` hash wouldn't be valid in the translated page. Hmm... not sure how to handle that.

File: website/pages/en/cookbook/subgrounds-quickstart.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+```python
+# Construct your query
+top_collections = seaport.Query.collections(
+    first=1500,
+    orderBy=seaport.Collection.totalRevenueETH,
+    orderDirection='desc',
+)
+```
+
+Note that with subgrounds, enables going beyond [the limit](https://thegraph.com/docs/en/developing/developer-faqs/#23-is-there-a-limit-to-how-many-objects-the-graph-can-return-per-query) of 1,000 items per collection without any need for manual intervention. By default, subgrounds handles GraphQL query pagination automatically, enabling effortless retrieval of large datasets.
```
Comment (benface) [inline website/pages/en/cookbook/subgrounds-quickstart.mdx:71]:
```suggestion
Note that with subgrounds, enables going beyond [the limit](/developing/developer-faqs/#23-is-there-a-limit-to-how-many-objects-the-graph-can-return-per-query) of 1,000 items per collection without any need for manual intervention. By default, subgrounds handles GraphQL query pagination automatically, enabling effortless retrieval of large datasets.
```

Comment (marcusrein) [reply website/pages/en/cookbook/subgrounds-quickstart.mdx:71]: The Playgrounds team was okay not integrating these docs updates into production as there is already a quckstart for Subgrounds in the docs.

**Pull request closed without merging**